README.md

```diff
@@ -87,6 +87,11 @@ DocRaptor.download(download_key)
 
 The `download` function works like `DocRaptor.create` in that you get back either an HTTParty response object or you can give it a block.
 
+
+## Privacy
+
+By default, this library will send DocRaptor information about the version of the gem you are using as well as your ruby runtime and ruby version. If you would like to not report that data, you can do so by calling `DocRaptor.disable_agent_tracking` when you initialize the library.
```

Comment:
Let's explain why we want to collect this information, how about:

To help us improve and support DocRaptor, this library will send DocRaptor information about the version of the gem you are using, as well as your ruby runtime. If you would like to not report that data, you can do so by calling `DocRaptor.disable_agent_tracking` when you initialize the library.